streaming/base/storage/download.py

```diff
@@ -47,14 +48,23 @@ def _download_file(unsigned: bool = False) -> None:
     if obj.scheme != 's3':
         raise ValueError(f'Expected obj.scheme to be "s3", got {obj.scheme} for remote={remote}')
 
+    extra_args = {}
+    # When enabled, the requester instead of the bucket owner pays the cost of the request
+    # and the data download from the bucket.
+    if os.environ.get('MOSAICML_STREAMING_AWS_REQUESTER_PAYS') == '1':
```

Comment:
We need to specify the list of bucket names

Reply:
Done in https://github.com/mosaicml/streaming/pull/231/commits/6a80cc6d97a762dea107a75b4c6247fff5058ced.